content/blog/2025-10-21-announcing-btrfs-support-in-almalinux-10-1.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Announcing Btrfs support in AlmaLinux OS 10.1"
+type: blog
+author:
+  name: "Davide Cavalca"
+  bio: "Production Engineer, Linux Userspace at Meta"
+  image: /users/davidecavalca.png
+date: "2025-10-21"
+images:
+  - /blog-images/2025/btrfs_almalinux101.png
+post:
+  title: "Announcing Btrfs support in AlmaLinux OS 10.1"
+  image: /blog-images/2025/btrfs_almalinux101.png
+---
+
+# Announcing Btrfs support in AlmaLinux OS 10.1
+
+AlmaLinux OS 10.1 will support the Btrfs filesystem, which has already been available in AlmaLinux OS Kitten since 2025-09-10.
```

Comment:
```suggestion
AlmaLinux OS 10.1 will support the Btrfs filesystem, which has already been available in AlmaLinux OS Kitten since early September.
```
Dates are always confusing for readers, so let's just reference the month.